src/Collector/SuperGlobalsCollector.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SensioLabs\Deptrac\Collector;
+
+use SensioLabs\Deptrac\AstRunner\AstMap;
+use SensioLabs\Deptrac\AstRunner\AstMap\AstClassReference;
+
+final class SuperGlobalsCollector implements CollectorInterface
+{
+    public function getType(): string
+    {
+        return 'superGlobals';
+    }
+
+    public function satisfy(array $configuration, AstClassReference $astClassReference, AstMap $astMap, Registry $collectorRegistry): bool
```

Comment:
I think the configuration of this collector should support that the user can choose which super-global should be matched.

e.g. I can forsee that e.g. the usage of `$_SERVER` and `$_ENV` would be more welcome in certain layers of the app then a  `$_GET/_POST/REQUEST/_COOKIE` etc.